test-suite/non-functional-tests/test-scenario/README.md

```diff
@@ -1,31 +1,59 @@
-## Test Scenario 6
-
-120 simultaneous transfers
+With this test plan it is possible to run the below scenarios:
 
-### Test details
+## Each Sucessful transfer contains
 
-120 Transfers requested - 100 expected to succeed, 20 expected to fail, each successful transfer contains:
 * 1 RCMR_IN030000UK06 message
 * 1 COPC_IN000001UK01 index message
 * 2 COPC_IN000001UK01 Fragment messages
 
-### Prerequisites 
+## Test Scenario 1
```

Comment:
Nice clean up of the different folders.

I think it would be useful to know in a bit more detail how to specifically set up these scenarios using the JMeter file. Perhaps saying which `properties` to set with each value?

<pre>
## Test Scenario 1

12 patient transfers x 1 thread

12 Transfers requested - 10 expected to succeed, 2 expected to fail. 

```
thing = 1
blah = 100
```
</pre>